assets/css/main.css

```diff
@@ -2434,7 +2433,17 @@
 					}
 
 			}
+			@media screen and (min-width: 736px) {
+				.features > li.project {
```

Comment:
Is this passing in the values of `.features` defined above into the `li.project`?

Reply:
No, it is a selector that only selects direct children of elements with class features that are of class project and of element type `li`. The `>` instead of a space makes it so that it only selects direct children rather than any children